### ISSUE TITLE
Adoption Insights plugin: Remove accessibility violation filter and fix remaining a11y issues

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3577]
The adaption insights plugins has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/adoption-insights/e2e-tests/utils/accessibility.ts`, but some violations was ignored at the moment. The are filtered out.

1. Remove the accessibility violation filter incl. the reference to ticket RHDHBUGS-2216.
2. Run the playwright tests via `CI=1 yarn test:e2e` and list the left over issues.
3. Try to resolve these issues in the code base. When unsure, or not all issues can be removed, create a PR anyway.

**Comment by its-mitesh-kumar** [comment]
Closing this in favor of https://github.com/redhat-developer/rhdh-plugins/issues/3835
